postgres://hppoc:password@127.0.0.1:5432/fabricexplorer

Property config.syncStartDate set to  2018/07/10
events.js:183
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE :::9090
    at Object._errnoException (util.js:1022:11)
    at _exceptionWithHostPort (util.js:1044:20)
    at Server.setupListenHandle [as _listen2] (net.js:1367:14)
    at listenInCluster (net.js:1408:12)
    at Server.listen (net.js:1492:7)
    at startExplorer (/home/sathiyan/Projects/fabric/captive-insurance_blockchain_explorer/main.js:58:10)
    at <anonymous>
postgres://hppoc:password@127.0.0.1:5432/fabricexplorer

Property config.syncStartDate set to  2018/07/10


Please open web browser to access ：http://localhost:9090/


pid is 19331


Insert sql is INSERT INTO orderer  ( "requests","server_hostname" ) VALUES( $1,$2  ) RETURNING *;
/home/rapidqube/Kare4U-Explorer/parametericInsurance__blockchain_explorer/app/persistence/postgreSQL/db/pgservice.js:319
      if (!res.rows || res.rows.length == 0) resolve(null);
               ^

TypeError: Cannot read property 'rows' of undefined
    at Query.client.query [as callback] (/home/rapidqube/Kare4U-Explorer/parametericInsurance__blockchain_explorer/app/persistence/postgreSQL/db/pgservice.js:319:16)
    at Query.handleError (/home/rapidqube/Kare4U-Explorer/parametericInsurance__blockchain_explorer/node_modules/pg/lib/query.js:143:17)
    at Connection.connectedErrorHandler (/home/rapidqube/Kare4U-Explorer/parametericInsurance__blockchain_explorer/node_modules/pg/lib/client.js:132:26)
    at emitOne (events.js:116:13)
    at Connection.emit (events.js:211:7)
    at Socket.<anonymous> (/home/rapidqube/Kare4U-Explorer/parametericInsurance__blockchain_explorer/node_modules/pg/lib/connection.js:117:12)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
postgres://hppoc:password@127.0.0.1:5432/fabricexplorer
(node:21998) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, scandir '../parametericInsurance_backend/basic-network/crypto-config/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore'
    at Object.fs.readdirSync (fs.js:904:18)
    at Object.readAllFiles (/home/rapidqube/explorer/parametericInsurance__blockchain_explorer/app/helper.js:25:18)
    at Platform.setAdminForClient (/home/rapidqube/explorer/parametericInsurance__blockchain_explorer/app/platform/fabric/Platform.js:78:37)
    at Platform.initialize (/home/rapidqube/explorer/parametericInsurance__blockchain_explorer/app/platform/fabric/Platform.js:146:18)
    at <anonymous>
(node:21998) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)
(node:21998) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
postgres://hppoc:password@127.0.0.1:5432/fabricexplorer

Property config.syncStartDate set to  2018/07/10


Please open web browser to access ：http://localhost:9090/


pid is 29730


/home/rapidqube/explorer/parametericInsurance__blockchain_explorer/app/persistence/postgreSQL/db/pgservice.js:319
      if (!res.rows || res.rows.length == 0) resolve(null);
               ^

TypeError: Cannot read property 'rows' of undefined
    at Query.client.query [as callback] (/home/rapidqube/explorer/parametericInsurance__blockchain_explorer/app/persistence/postgreSQL/db/pgservice.js:319:16)
    at Query.handleError (/home/rapidqube/explorer/parametericInsurance__blockchain_explorer/node_modules/pg/lib/query.js:143:17)
    at Connection.connectedErrorHandler (/home/rapidqube/explorer/parametericInsurance__blockchain_explorer/node_modules/pg/lib/client.js:132:26)
    at emitOne (events.js:116:13)
    at Connection.emit (events.js:211:7)
    at Socket.<anonymous> (/home/rapidqube/explorer/parametericInsurance__blockchain_explorer/node_modules/pg/lib/connection.js:117:12)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
Received kill signal, shutting down gracefully
Closed out remaining connections
postgres://hppoc:password@127.0.0.1:5432/fabricexplorer

Property config.syncStartDate set to  2018/07/10


Please open web browser to access ：http://localhost:9090/


pid is 3191


/home/rapidqube/explorer/parametericInsurance__blockchain_explorer/app/persistence/postgreSQL/db/pgservice.js:464
      if (!res.rows || res.rows.length == 0) resolve(null);
               ^

TypeError: Cannot read property 'rows' of undefined
    at Query.client.query [as callback] (/home/rapidqube/explorer/parametericInsurance__blockchain_explorer/app/persistence/postgreSQL/db/pgservice.js:464:16)
    at Query.handleError (/home/rapidqube/explorer/parametericInsurance__blockchain_explorer/node_modules/pg/lib/query.js:143:17)
    at Connection.connectedErrorHandler (/home/rapidqube/explorer/parametericInsurance__blockchain_explorer/node_modules/pg/lib/client.js:132:26)
    at emitOne (events.js:116:13)
    at Connection.emit (events.js:211:7)
    at Socket.<anonymous> (/home/rapidqube/explorer/parametericInsurance__blockchain_explorer/node_modules/pg/lib/connection.js:117:12)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
